example-doc-win2019.md

```diff
@@ -0,0 +1,261 @@
+- Microsoft.VisualStudio.Component.VC.MFC.ARM64
+- Microsoft.VisualStudio.Component.VC.MFC.ARM64.Spectre
+- Microsoft.VisualStudio.Component.VC.Redist.MSM
+- Microsoft.VisualStudio.Component.VC.Runtimes.ARM.Spectre
+- Microsoft.VisualStudio.Component.VC.Runtimes.ARM64.Spectre
+- Microsoft.VisualStudio.Component.VC.Runtimes.x86.x64.Spectre
+- Microsoft.VisualStudio.Component.VC.TestAdapterForBoostTest
+- Microsoft.VisualStudio.Component.VC.TestAdapterForGoogleTest
+- Microsoft.VisualStudio.Component.VC.v141
+- Microsoft.VisualStudio.Component.VC.v141.x86.x64
+- Microsoft.VisualStudio.Component.VC.v141.x86.x64.Spectre
+- Microsoft.VisualStudio.Component.VC.v141.ARM.Spectre
+- Microsoft.VisualStudio.Component.VC.v141.ARM64.Spectre
+- Microsoft.VisualStudio.Component.VC.v141.ATL.ARM.Spectre
+- Microsoft.VisualStudio.Component.VC.v141.ATL.ARM64.Spectre
+- Microsoft.VisualStudio.Component.VC.v141.ATL.Spectre
+- Microsoft.VisualStudio.Component.VC.v141.MFC.ARM.Spectre
+- Microsoft.VisualStudio.Component.VC.v141.MFC.ARM64.Spectre
+- Microsoft.VisualStudio.Component.VC.v141.MFC.Spectre
+- Microsoft.VisualStudio.Component.Windows10SDK.14393
+- Microsoft.VisualStudio.Component.Windows10SDK.16299
+- Microsoft.VisualStudio.Component.Windows10SDK.17134
+- Microsoft.VisualStudio.Component.Windows10SDK.17763
+- Microsoft.VisualStudio.Component.Windows10SDK.18362
+- Microsoft.VisualStudio.Component.WinXP
+- Microsoft.VisualStudio.ComponentGroup.Azure.CloudServices
+- Microsoft.VisualStudio.ComponentGroup.Azure.ResourceManager.Tools
+- Microsoft.VisualStudio.ComponentGroup.NativeDesktop.Llvm.Clang
+- Microsoft.VisualStudio.ComponentGroup.Web.CloudTools
+- Microsoft.VisualStudio.Workload.Azure
+- Microsoft.VisualStudio.Workload.Data
+- Microsoft.VisualStudio.Workload.DataScience
+- Microsoft.VisualStudio.Workload.ManagedDesktop
+- Microsoft.VisualStudio.Workload.ManagedGame
+- Microsoft.VisualStudio.Workload.NativeCrossPlat
+- Microsoft.VisualStudio.Workload.NativeDesktop
+- Microsoft.VisualStudio.Workload.NativeGame
+- Microsoft.VisualStudio.Workload.NativeMobile
+- Microsoft.VisualStudio.Workload.NetCoreTools
+- Microsoft.VisualStudio.Workload.NetCrossPlat
+- Microsoft.VisualStudio.Workload.NetWeb
+- Microsoft.VisualStudio.Workload.Node
+- Microsoft.VisualStudio.Workload.Office
+- Microsoft.VisualStudio.Workload.Python
+- Microsoft.VisualStudio.Workload.Universal
+- Microsoft.VisualStudio.Workload.VisualStudioExtension
+
+#### Microsoft SSDT Visual Studio 2019 Extensions
+- Microsoft Analysis Services Projects Version: 2.9.8
+- SQL Server Integration Services Projects Version: 3.6
+- Microsoft Reporting Services Projects Version: 2.6.5
+
+#### Windows Driver Kit
+- WDK Version: 10.1.18362.1
+- WDK Visual Studio Extension Version: 10.0.18346.0
+
+#### WIX Tools
+- Toolset Version: 3.11.4516
+- WIX Toolset Studio 2019 Extension Version: 1.0.0.4
+
+
+### .NET Core
+- SDK: 2.1.300, 2.1.301,... , 3.1.200, 3.1.201
+- SDK Path: C:\Program Files\dotnet\sdk
```

Comment:
Choose another PATH separator

Reply:
Maybe also convert this section to table? For example
![image](https://user-images.githubusercontent.com/8777209/81715498-f7a91700-9480-11ea-86ac-e5e41fe2dfd9.png)